frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -1,117 +1,129 @@
-      Record<string, any>,
-      Record<string, any>
-    >
-  >;
 }
 
 const TableInstance: FC<ITableInstanceProps> = ({ ...props }) => {
-  const {
-    instances,
-    viewMode,
-    extended,
-    showGuiIcon,
-    startInstance,
-    stopInstance,
-    destroyInstance,
-  } = props;
+  const { instances, viewMode, extended, showGuiIcon } = props;
 
   const [now, setNow] = useState(new Date());
+  const [deleteInstanceMutation] = useDeleteInstanceMutation();
 
   setInterval(() => setNow(new Date()), 60000);
 
+  const destroyAll = () => {
+    instances.forEach(instance => {
+      deleteInstanceMutation({
+        variables: {
+          instanceId: instance.name,
+          tenantNamespace: instance.tenantNamespace!,
+        },
+      });
+    });
+  };
+
   const data = instances;
   return (
-    <div
-      className={`rowInstance-bg-color ${
-        viewMode === 'user' && extended
-          ? 'cl-table-instance flex-grow flex-wrap content-between py-0 overflow-auto scrollbar'
-          : ''
-      }`}
-    >
-      <Table
-        className="rowInstance-bg-color"
-        dataSource={data}
-        showHeader={false}
-        pagination={false}
-        size={'middle'}
-        rowClassName={
-          viewMode === 'user' && extended ? '' : 'rowInstance-bg-color'
-        }
-        rowKey={record =>
-          extended && viewMode === 'user'
-            ? record.id + record.idTemplate!
-            : record.id
-        }
+    <>
+      <div
+        className={`rowInstance-bg-color ${
+          viewMode === 'user' && extended
+            ? 'cl-table-instance flex-grow flex-wrap content-between py-0 overflow-auto scrollbar'
+            : ''
+        }`}
       >
-        <Column
-          className={
-            extended
-              ? viewMode === WorkspaceRole.user
-                ? 'w-5/6 sm:w-2/3 lg:w-3/5 xl:w-1/2 2xl:w-5/12'
-                : 'w-3/5 md:w-1/2'
-              : 'w-2/3 md:w-3/4'
+        <Table
+          className="rowInstance-bg-color"
+          dataSource={data}
+          showHeader={false}
+          pagination={false}
+          size={'middle'}
+          rowClassName={
+            viewMode === 'user' && extended ? '' : 'rowInstance-bg-color'
           }
-          title="Instance Title"
-          key="title"
-          render={(instance: Instance) => (
-            <RowInstanceTitle
-              viewMode={viewMode}
-              extended={extended}
-              instance={instance}
-              showGuiIcon={showGuiIcon}
-            />
-          )}
-        />
-        <Column
-          className={
-            extended
-              ? viewMode === WorkspaceRole.user
-                ? 'w-1/6 sm:w-1/3 lg:w-2/5 xl:w-1/22xl:w-7/12'
-                : 'w-2/5 md:w-1/2'
-              : 'w-1/3 md:w-1/4'
+          rowKey={record =>
+            extended && viewMode === 'user'
+              ? record.id + record.idTemplate!
+              : record.id
           }
```

Comment:
```suggestion
          rowKey={record =>record.id + record.idTemplate!}
```
Can't this work?

Reply:
i think it was something old, now it should work as you suggest